autoprotocol/harness.py

```diff
@@ -26,6 +26,23 @@
 }
 
 
+def get_protocol_preview(protocol, name):
+    manifest_json = io.open('manifest.json', encoding='utf-8').read()
+    manifest = Manifest(json.loads(manifest_json))
+
+    try:
+        source = [
+            manifest.protocols[i] for i, m in enumerate(manifest.protocols)
+            if manifest.protocols[i]['name'] == name][0]['preview']
```

Comment:
nit: the formatting is slightly odd here. there's also an implicit picking of the first match, which would be good to call out/document or add an error case for
```
protocols = [
       manifest.protocols[i] for i, m in enumerate(manifest.protocols)
       if manifest.protocols[i]['name'] == name
]
source = protocols[0]['preview'] # or some other check
```